Content.Server/Fluids/Components/SinkComponent.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Robust.Shared.Audio;
+namespace Content.Server.Fluids.Components;
+
+/// <summary>
+/// Allows an entity to absorb any amount of liquid from a container (beaker etc)
+/// </summary>
+[RegisterComponent]
+public sealed class SinkComponent : Component
```

Comment:
access

Reply:
Can you elaborate? I don't get what you mean